sale_quotation_number/models/sale_order.py

```diff
@@ -0,0 +1,52 @@
+# © 2010-2012 Andy Lu <andy.lu@elico-corp.com> (Elico Corp)
+# © 2013 Agile Business Group sagl (<http://www.agilebg.com>)
+# © 2017 valentin vinagre  <valentin.vinagre@qubiq.es> (QubiQ)
+# © 2020 Manuel Regidor  <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            if self.is_using_quotation_number(vals):
+                sequence = self.env["ir.sequence"].next_by_code("sale.quotation")
+                vals["name"] = sequence or "/"
+            return super().create(vals)
```

Comment:
The return statement must be out of the for loop.
I think that you can just modify the values inside the loop and after call super() with vals_list.
Something like this:

```
    @api.model_create_multi
    def create(self, vals_list):
        for vals in vals_list:
            if self.is_using_quotation_number(vals):
                sequence = self.env["ir.sequence"].next_by_code("sale.quotation")
                vals["name"] = sequence or "/"
        return super().create(vals_list)
```

Reply:
Hi @dariodelzozzo Done the changes